library/core/resource-loader/src/main/java/org/quiltmc/qsl/resource/loader/mixin/DefaultResourcePackMixin.java

```diff
@@ -45,11 +45,11 @@
 public abstract class DefaultResourcePackMixin {
 	// Redirects all resource access to the MC resource pack.
 	@Unique
-	final ModNioResourcePack internalPack = this.locateAndLoad();
+	final ModNioResourcePack quilt$internalPack = this.quilt$locateAndLoad();
 
 	@SuppressWarnings({"ConstantConditions", "EqualsBetweenInconvertibleTypes"})
 	@Unique
-	private ModNioResourcePack locateAndLoad() {
+	private ModNioResourcePack quilt$locateAndLoad() {
```

Comment:
I believe mixin will still scrambe these private methods, enable the export environment variable to see further

Reply:
Did you confirm that it doesn't scramble the name in a conflict? The javadoc is very clear that it will, so having to prefix this method would be a bug.
> non-public methods are **renamed** [emphasis original] if a matching target method is
> found, this allows utility methods to be safely assigned meaningful names
 > in code, but renamed if a conflict occurs when a mixin is applied.

Reply:
It only renames if such method/field exists in the final class as part of the original code-base.

This PR was motivated after seeing those screenshots in the Quilt Community Discord.
![image](https://user-images.githubusercontent.com/12587332/129762674-8d987c6e-9b62-479f-906c-51345702d05d.png)
![image](https://user-images.githubusercontent.com/12587332/129762695-81a7c973-e63d-4e61-b58c-b2c932157578.png)

Reply:
Ok, so, private methods do get scrambled, but fields don't.